### PR TITLE
Update footer.html for Facebook auth

### PR DESCRIPTION
**Pull request by davelab6**
In order to verify the facebook page, we need to have a confirmation number publicly listed on the homepage. This is my Google Voice, and once the fb page is verified, I'll remove it :)

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
   CC BY NC 4.0</a>
   </p>
 
-  <p>Copyright 2009-2015 Software Freedom Conservancy, Sugar Labs®</p>
+  <p>Copyright &copy;	2009-2015 Software Freedom Conservancy, Sugar Labs&reg;</p>
```
Comment (venky18) [inline _includes/footer.html]: shouldn't r be 2009-2016

**Comment by davelab6** [comment]
Thanks @venky18 :) fixed

**Comment by davelab6** [comment]
I suggest this can be merged.

**Comment by davelab6** [comment]
Samson pinged me about this today - please merge

**Comment by davelab6** [comment]
@samdroid-apps gave me commit access so I've merged this

**Comment by samswag** [comment]
nice on guys, let do the same for twitter

**Pull request merged**